packages/e2e-tests/test/e2e.spec.ts

```diff
@@ -735,6 +735,17 @@ describe('e2e', function () {
       );
     });
 
+    it('throws the .toArray() suggestion when a user attempts to serialize a cursor', async function () {
+      await shell.executeLine('const b = db.test.find()');
+      await shell.executeLine('b.b = b');
+      await shell.executeLine('console.log(JSON.stringify(b));');
+      await eventually(() => {
```

Comment:
In this case, the return value of `executeLine()` should already contain this output, so there's no need to use a less exact method like `eventually()`